storage/range_data_iter.go

```diff
@@ -136,3 +136,11 @@ func (ri *rangeDataIterator) advance() {
 		}
 	}
 }
+
+func (r *rangeDataIterator) SeekReverse(key []byte) {
+	panic("TODO(thundercw): unimplemented")
```

Comment:
I think we'll never want to reverse this iterator. Instead of a TODO, make this:

`panic("cannot reverse scan rangeDataIterator")`